### PR TITLE
revert on-commit

### PR DESCRIPTION
**Pull request by maskarb**
## Jira Ticket

[COST-####](https://issues.redhat.com/browse/COST-####)

## Description

This change will ...

## Testing

1. Checkout Branch
2. Restart Koku
3. Hit endpoint or launch shell
    1. You should see ...
4. Do more things...

## Notes

...

### CODE REVIEW

File: koku/api/provider/provider_manager.py [hunk 1]
```diff
@@ -272,17 +273,13 @@ def remove(self, request=None, user=None, from_sources=False, retry_count=None):
                 self.model.delete()
                 LOG.info(f"Provider: {self.model.name} removed by {current_user.username}")
             except IntegrityError as err:
-                # One more place to retryfor when we delete smoke test sources while they have
-                # data processing
-                if retry_count is not None and retry_count < settings.MAX_SOURCE_DELETE_RETRIES:
-                    err_msg = f"Provider {self._uuid} is currently being processed and must finish before delete."
-                    raise ProviderProcessingError(err_msg)
-                else:
+                if retry_count is None or retry_count >= settings.MAX_SOURCE_DELETE_RETRIES:
```
Comment (adberglund) [inline koku/api/provider/provider_manager.py:276]: 🌮

**Comment by maskarb** [comment]
We originally opened this thinking some OperationalErrors were caused by these original changes, but haven't been able to verify that these changes were the cause. Closing this PR.

**Pull request closed without merging**